src/server/schema.ts

```diff
@@ -33,10 +33,20 @@ export const schema = `
     terms: [Term]! @deprecated(reason: "Use 'result' instead")
     result: TermsResult!
   }
+  
+# TBD: Alternative result type.
+#  type LookupResult {
+#    uri: ID!
+#    source: Source
+#    result: LookupSuccessErrorResult
+#  }
```

Comment:
TODO: discuss

Reply:
True. Or should we keep the `uri`, to allow API users to get to the right term faster? If they provide multiple URIs to the `lookup` endpoint, they need an easy way to find the metadata that belongs to each URI.

Reply:
Yes, I kept the URI. This is especially relevant when we can find no term for the URI, in which case `Term.uri` is not available.